composer.json

```diff
@@ -23,12 +23,17 @@
         "symfony/phpunit-bridge": "~2.7|~3.0"
     },
 
+    "require-dev": {
+        "symfony/http-kernel": "^2.3 || ^3.0"
```

Comment:
Adding a dev requirement makes it easier to work on fixtures, as the IDE will have the code available.